CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
```

Comment:
I feel like this puts the barrier for contributions a bit too high.
Simple contributions should be possible with just a PR, if they need further discussions this can always be done in the PR or split out into follow-up issues.

Reply:
I rephrase it. I hope it's a bit more welcoming.